code/obj/stool.dm

```diff
@@ -913,6 +936,12 @@
 			C.set_dir(dir)
 		return
 
+	attackby(obj/item/W as obj, mob/user as mob)
+		var/butttest = has_butt
+		..()
+		if (has_butt != butttest)
```

Comment:
II don't think that this does what you want it to, but then I can't really tell what that is. Did you test this code to make sure it works?

I think you'd want just a null check on the has_butt var after the ..() to check if you need to update? But even then I'm not sure why you need to do that.

Reply:
The point of this bit was to make the chair update the arm overlay so it layers over the butt, though it can probably be done simpler? IIRC it did work in testing for that purpose.

Will look at it tomorrow

Reply:
Oh. If that's the intent, then why not just default the layer of the butt overlay to a value in between the chair's layer and the arm's layer?

Reply:
Done